### DISCUSSION TITLE
Tailwind idea

### DISCUSSION THREAD

**Discussion by nam-co** [bdkjones/CodeKit#711] [Ideas]
I has resistant to tailwind, but after a few years and features like **@apply**, tailwind is making a lot of sense, still I feel that is a pain to set it up, specially for quick/small sites, so @bdkjones just an idea:

A built-in tailwind (no node_modules) you just create a scss or css file and add something like // @codekit-tailwind "tailwind.config.js"; or just a checkbox in the app "Use Tailwind" and then TW is automatically included when the file is process (force us to renew just to get the new tailwind Version)

This could be cool

**Comment by bdkjones** [comment]
Tailwind is already built-in. File > New Tailwind Project.

The project structure is important for Tailwind, so it's not possible to just add it to any existing Project.
